### ISSUE TITLE
Full preview does not work in Lighttable

### ISSUE THREAD

**Issue by olmet** [aurelienpierreeng/ansel#155]
Fully-zoomed preview does not work in the Lighttable module. 

When I pressing and holding Ctrl+W or F appears: "F not assigned". There is no entry for Full Preview in the Lighttable module in the actions in the keyboard shortcuts. How can I add this action?

Thanks and sorry for my bad English.

version: 0.0.0+285~g0c37799cc

**Comment by l-28** [comment]
Hey @olmet,

the full-screen preview is unfortunately not available at the moment, because the Lighttable view is under reconstruction. So in a way this is to be expected and you might close this issue.

You will find that you are not alone with this problem as in this similar report: [Can't find the zoom options that were triggered by pressing W and Ctrl+W in lighttable](https://community.ansel.photos/view-discussion/can-t-find-the-zoom-options-that-were)

and that there is a workaround for the time being: [Culling/Rating Images Workflow](https://community.ansel.photos/view-discussion/culling-rating-images-workflow)

Hope that helps

**Comment by olmet** [comment]
Hallo @l-28 
Thanks for your answer, I will try the workaround for now.

**Comment by barathrm** [comment]
Should this issue not be kept open until it is fixed or decided not to be fixed?

**Comment by olmet** [comment]
You are right, the workaround does not fix the problem. I keep this issue open.

**Comment by aurelienpierre** [comment]
Lighttable got rewritten from scratch. Full preview can be achieved by setting 1 column per row (so thumbnails take the whole space), hiding the thumbnail overlays. You can zoom at 50 or 100% of the full-resolution of the original image too.
